modules/apps/transfer/keeper/keeper_test.go

```diff
@@ -44,3 +48,139 @@ func NewTransferPath(chainA, chainB *ibctesting.TestChain) *ibctesting.Path {
 func TestKeeperTestSuite(t *testing.T) {
 	suite.Run(t, new(KeeperTestSuite))
 }
+
+func (suite *KeeperTestSuite) TestSetGetTotalEscrowForDenom() {
+	var expAmount math.Int
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success: with escrow amount > 2^63",
+			func() {
+				expAmount, _ = math.NewIntFromString("100000000000000000000")
+			},
+			true,
+		},
+		{
+			"failure: setter panics with negative escrow amount",
+			func() {
+				expAmount = math.NewInt(-1)
+			},
+			false,
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+
+		suite.Run(tc.name, func() {
+			suite.SetupTest() // reset
+			expAmount = math.ZeroInt()
+			ctx := suite.chainA.GetContext()
+
+			tc.malleate()
+
+			if tc.expPass {
+				suite.chainA.GetSimApp().TransferKeeper.SetTotalEscrowForDenom(ctx, "atom", expAmount)
+				total := suite.chainA.GetSimApp().TransferKeeper.GetTotalEscrowForDenom(ctx, "atom")
+				suite.Require().Equal(expAmount, total)
+			} else {
+				suite.Require().PanicsWithValue("amount cannot be negative: -1", func() {
+					suite.chainA.GetSimApp().TransferKeeper.SetTotalEscrowForDenom(ctx, "atom", expAmount)
+				})
+				total := suite.chainA.GetSimApp().TransferKeeper.GetTotalEscrowForDenom(ctx, "atom")
+				suite.Require().Equal(math.ZeroInt(), total)
+			}
+		})
+	}
+}
+
+func (suite *KeeperTestSuite) TestGetAllDenomEscrows() {
```

Comment:
Could we add a test for many kv mappings since this function performs iteration?